src/components/App.js

```diff
@@ -8,6 +8,7 @@ import Todo from 'components/Todo/Todo'
 import Button from 'components/Button/Button'
 
 import todosList from 'data'
+import { BsTrash } from 'react-icons/bs'
```

Comment:
Otra sugerencia, podés aprovechar los _alias_ para darle nombres más semánticos a los íconos y que sea más legible en el jsx

```suggestion
import { BsTrash as IconDelete } from 'react-icons/bs'
```